pkgs/development/python-modules/pypdfium2/default.nix

```diff
@@ -104,6 +118,11 @@ buildPythonPackage rec {
     '';
   env.PDFIUM_PLATFORM = "system:${pdfiumVersion}";
 
+  # Remove references to stdenv in comments.
+  postInstall = ''
+    remove-references-to -t ${stdenv.cc.cc} $out/${python.sitePackages}/pypdfium2_raw/bindings.py
```

Comment:
i notice native builds also have references to `glibc.dev` in comments in this same file. if you're feeling curious, could be worth looking into (does cross not have those references because it's not building all the same bindings as native builds?); maybe adding `-t ${glibc.dev}` here too. hardly a blocker though.

Reply:
The bindings are the same, it just uses gcc's `sys-include/bits/types.h` for cross and the same in glibc.dev for native. I'll make a follow-up PR, thanks for checking.

Reply:
Wondering what it would use on musl :thinking: